state/machine.go

```diff
@@ -721,7 +721,7 @@ func (original *Machine) advanceLifecycle(life Life, force bool, maxWait time.Du
 	}
 	// noUnits asserts that the machine has no principal units.
 	noUnits := bson.DocElem{
-		"$or", []bson.D{
+		Name: "$or", Value: []bson.D{
```

Comment:
These changes are in a separate commit. They are all lint fixes and this file can be ignored.